### DISCUSSION TITLE
제페토 카메라 초기 위치, 회전 설정

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#676] [5. Q&A]
월드 1.2.0 사용중입니다 

혹시 플레이어가 로드 되었을때 제페토 카메라의 위치와 회전을 원하는 값으로 설정 할수 있나요?
![Screen Shot 2022-04-28 at 오후 6 25](https://user-images.githubusercontent.com/50188795/165722199-dd42f445-8024-4e91-a978-8542bba512e4.png)

**Comment by minjujuu** [comment]
아래와 같은 코드로 하면 캐릭터가 로드된 시점에 카메라의 회전 값을 변경할 수 있습니다.

```
            // [CharacterController] (Local)Player 인스턴스가 Scene에 완전히 로드되었을 때 호출
            ZepetoPlayers.instance.OnAddedLocalPlayer.AddListener(() => {
                ZepetoPlayers.instance.ZepetoCamera.transform.rotation = UnityEngine.Quaternion.Euler(new UnityEngine.Vector3(0, 90, 0));
            });
```

다만 위치값은 ZepetoPlayers에 있는 카메라 관련 옵션 값의 여러 영향을 받고, 
매 프레임마다 캐릭터를 타겟으로 하도록 설정되어 있는 듯 해서 
다른 위치값을 적용하고 싶다면 제페토 카메라가 아닌 새로운 카메라를 만들어서 사용하시는 게 좋을 것 같습니다!
